src/aiod_sdk/endpoints/counts.py

```diff
@@ -2,75 +2,54 @@
 import requests
 from typing import Literal
 
-from aiod_sdk.endpoints.endpoint import EndpointBase
+from aiod_sdk.endpoints.settings import api_base_url, latest_version
 
 
-class Counts(EndpointBase):
+def asset_counts(
+    version: str | None = None, format: Literal["pandas", "dict"] = "pandas"
+) -> pd.DataFrame | dict:
     """
-    Class for retrieving counts of resources.
+    Retrieve counts of assets.
 
-    This class provides a method to retrieve counts of resources from the counts endpoint.
+    This method sends a GET request to the counts endpoint to retrieve counts of assets.
 
-    Inherits From:
-        EndpointBase: Provides base functionality for endpoint interactions.
+    Parameters:
+        version (str | None): The version of the counts endpoint (default is None).
+        format (Literal["pandas", "dict"]): The desired format for the response (default is "pandas").
 
-    Attributes:
-        name (str): The name of the counts endpoint.
-        latest_version (str): The latest version of the counts endpoint.
-
-    Methods:
-        asset_counts(version=None) -> requests.Response:
-            Retrieve counts of assets.
+    Returns:
+        pd.DataFrame | dict: Counts as a Pandas data frame or a dictionary.
     """
+    version = version if version is not None else latest_version
+    url = api_base_url + "counts"
+    if version:
+        url += "/" + version
 
-    name = "counts"
-
-    @staticmethod
-    def _format_response(
-        response: list | dict, format: Literal["pandas", "dict"]
-    ) -> pd.DataFrame | dict:
-        """
-        Format the response data based on the specified format.
-
-        Parameters:
-            response (list | dict): The response data to format.
-            format (Literal["pandas", "dict"]): The desired format for the response.
-
-        Returns:
-            pd.DataFrame | dict: The formatted response data.
+    res = requests.get(url)
+    counts = _format_response(res.json(), format)
+    return counts
 
-        Raises:
-            Exception: If the specified format is invalid or not supported.
-        """
 
-        if format == "pandas":
-            return pd.DataFrame(response)
-        elif format == "dict":
-            return response
-        else:
-            raise Exception(f"Format: {format} invalid or not supported.")
-
-    @classmethod
-    def asset_counts(
-        cls, version: str | None = None, format: Literal["pandas", "dict"] = "pandas"
-    ) -> pd.DataFrame | dict:
-        """
-        Retrieve counts of assets.
+def _format_response(
```

Comment:
Import the function from `endpoints.calls`

Reply:
This function is slightly different than the other one, but I found out that its docstring was outdated as well. Since it is only used by the `asset_counts` method and it is pretty simple, I merged it to the `asset_counts` method. I replaced the "dict" option to "json". Explanation in the comment above. 
```python
...
res = requests.get(url)

if data_format == "pandas":
    return pd.DataFrame(res.json())

elif data_format == "json" and isinstance(res.json(), dict):
    return res.json()

raise Exception(
    f"Format: {data_format} invalid or not supported for responses of {type(res.json())=}."
)
```